doc/src/manual/noteworthy-differences.md

```diff
@@ -18,8 +18,8 @@ may trip up Julia users accustomed to MATLAB:
     which grow `Vector`s much more efficiently than MATLAB's `a(end+1) = val`.
   * The imaginary unit `sqrt(-1)` is represented in Julia as [`im`](@ref), not `i` or `j` as in MATLAB.
   * In Julia, literal numbers without a decimal point (such as `42`) create integers instead of floating
-    point numbers. Arbitrarily large integer literals are supported. As a result, some operations
-    such as `2^-1` will throw a domain error as the result is not an integer (see [the FAQ entry on domain errors](@ref faq-domain-errors)
+    point numbers. Arbitrarily large integer literals are supported, subject to the limitations of Julia's
```

Comment:
```suggestion
    point numbers. Arbitrarily large integer literals are supported.
```

Reply:
Ok, thanks for pointing this out. I will revert it.

Reply:
Actually, what do you think about dropping the point entirely? It seems like its covered in detail in the manual entry on julia integer support, and doesn't add any information about the main point here, which is that in julia you don't get a float when you write `1`.

Reply:
Sure, it does seem like a distraction from the main point.